tests/btb/test_hyper_parameter.py

```diff
@@ -13,15 +13,16 @@ def setUp(self):
         self.parameter_constructions = [
             (ParamTypes.INT, [1, 3]),
             (ParamTypes.INT_EXP, [10, 10000]),
+            (ParamTypes.INT_CAT, [10, 10000]),
             (ParamTypes.FLOAT, [1.5, 3.2]),
             (ParamTypes.FLOAT_EXP, [0.001, 100]),
             (ParamTypes.FLOAT_CAT, [0.1, 0.6, 0.5]),
             (ParamTypes.BOOL, [True, False]),
             (ParamTypes.STRING, ['a', 'b', 'c']),
         ]
 
-    def test___init__value_error(self):
-        with pytest.raises(ValueError):
+    def test_init_value_error(self):
```

Comment:
I think that the name `test___init__value_error` was right, following the convention `test_{method_name}_{case_description}`